extensions/amp-truncate-text/1.0/storybook/Basic.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../../src/preact';
+import {TruncateText} from '../truncate-text';
+import {withA11y} from '@storybook/addon-a11y';
+import {withKnobs} from '@storybook/addon-knobs';
+
+export default {
+  title: 'TruncateText',
+  component: TruncateText,
+  decorators: [withKnobs, withA11y],
+};
+
+export const SimpleTruncatedText = () => (
+  <truncate-text>
```

Comment:
Shouldn't this be `<TruncateText>`? Or did you change it on purpose since it's not working?

Reply:
Well that would certainly explain why this isn't working :man_facepalming: